src/Microsoft.DotNet.Wpf/src/WindowsBase/MS/Internal/IO/Packaging/CompoundFile/RightsManagementEncryptionTransform.cs

```diff
@@ -1138,16 +1138,12 @@ private static void
             bool validEnum = false;
 
             // user names: case-insensitive comparison
-            if (((IEqualityComparer) CU.StringCaseInsensitiveComparer).Equals(
-                    authenticationTypeString,
-                    Enum.GetName(typeof(AuthenticationType), AuthenticationType.Windows)))
+            if (string.Equals(authenticationTypeString, nameof(AuthenticationType.Windows)))
```

Comment:
I replaced the usage of `Enum.GetName` with `nameof` since we already know the enum value at build time so `Enum.GetName` is unnecessary overhead.